cluster/juju/layers/kubernetes-master/actions/create-rbd-pv

```diff
@@ -38,6 +38,14 @@ def main():
     this script thinks the environment is 'sane' enough to provision volumes.
     '''
 
+    # k8s >= 1.10 uses CSI and doesn't directly create persistent volumes
+    if get_version('kube-apiserver') >= (1, 10):
+       print('This action is deprecated in favor of CSI creation of persistent volumes')
+       print('in Kubernetes >= 1.10. Just create the PVC and a PV will be created')
+       print('for you.')
+       action_fail('Deprecated, just create PVC.')
+       return
```

Comment:
Can you update the action description too? https://github.com/hyperbolic2346/kubernetes-1/blob/bc3e25146f68c3f97ce9e2198831e4a0af1e47ff/cluster/juju/layers/kubernetes-master/actions.yaml#L4